### ISSUE TITLE
Clearer license/copyright statement

### ISSUE THREAD

**Issue by kbseah** [learn-teochew/learn-teochew.github.io#1]
Include license file in repo, and explicit copyright statement on index page detailing which parts are original content.

Consider CC-BY-ND or similar?

**Comment by programmeruser2** [comment]
Can I adapt the general structure of the website for another Chinese language? I'll rewrite all of the content, and I'll only keep the layout of the website.

**Comment by kbseah** [comment]
@programmeruser2 yes that's okay by me. the structure i used was influenced by Matthews and Yip's Cantonese grammar book, maybe you'd like to check it out. Happy to link to your site when it's done, too
